Packages/PrimitivesComponents/Sources/ViewModels/AssetScoreViewModel.swift

```diff
@@ -0,0 +1,52 @@
+// Copyright (c). Gem Wallet. All rights reserved.
+
+import Foundation
+import Primitives
+import Localization
+import GemstonePrimitives
+import Components
+import Style
+
+public enum AssetScoreViewModel {
+    case verified
+    case unverified
+    case malicious
+
+    public init(score: Int32) {
+        switch score {
+        case ...5:
+            self = .malicious
+        case 6...15:
+            self = .unverified
+        default:
+            self = .verified
+        }
+    }
+    
+    public var hasWarning: Bool {
```

Comment:
```suggestion
    public var hasWarning: Bool {
```

should be removed from here.

Reply:
`Asset` is responsible to decide if to show or not